backend/routes/notification.py

```
@@ -0,0 +1,34 @@
+
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+# Get unread notifications for a user
+@router.get("/unread", response_model=list[NotificationResponse])
+def fetch_unread_notifications(user_id: int, db: Session = Depends(get_db)):
```

Ruff (0.8.2)

Missing return type annotation for public function `fetch_unread_notifications`

(ANN201)